### PR TITLE
Address two 508 issues on patient builder page

### PR DESCRIPTION
**Pull request by lukoktonos**
This pull request is to address two 508 compliance issues on the patient builder page.
508 dictates that there should be only one <label for=""> for each input, and that each <label> tag be associated with an <input> tag.

1. Change "Relevant Period" orphaned label to be a span (labels should go with an input tag)

2. Change CQL Calculation results orphaned labels  to be spans, and adjust css to match previous style.

Pull requests into Bonnie require the following. (Reviewer should :heavy_check_mark:*when done*):

Submitter:

- [X] This pull request describes *why* these changes were made.
- [X] Tests are included and test edge cases N/A: Needs WAVE tool to test.
- [X] Link to JIRA test(s), if used: https://jira.mitre.org/browse/BONNIE-1111
Justification: JIRA test needed to use the WAVE chrome plugin tool.
- [X] JIRA test(s) have been added to sprint, if used 
- [X] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1021
- [X] JIRA ticket links to this PR.
- [X] Automated regression test(s) pass (if applicable): 

Reviewer 1:
- [x] The code is maintainable
- [x] The code reuses code or resources within the project ecosystem where applicable
- [x] The code achieves its purpose
- [x] The tests appropriately test edge cases
- [x] The tests pass

Reviewer 2: (Kristian)
- [X] The code is maintainable
- [X] The code reuses code or resources within the project ecosystem where applicable
- [X] The code achieves its purpose
- [x] The tests appropriately test edge cases
- [X] The tests pass

### CODE REVIEW

**Review by ssayer — DISMISSED** [review]
Latest commit fixes all known orphan label issues on Patient Builder.

File: app/assets/javascripts/templates/patient_builder/patient_builder.hbs [hunk 1]
```diff
@@ -104,8 +104,11 @@
               </div>
             </div>
           {{else}}
+            <span class="control-label">Living Status</span>
             <div class="checkbox deceased-checkbox">
+              <label for="expired">
                 <input type="checkbox" id="expired" name="expired"> Deceased
+              </label>
```
Comment (mulcahyk) [inline app/assets/javascripts/templates/patient_builder/patient_builder.hbs]: When I run the old code I'm not seeing anything in Wave for EXPIRED having multiple labels.
I took the test from this PR and added to the cql4bonnie branch and the test passed.  
I think that the issue with the EXPIRED labels was already fixed with the previous PR for 508 issues.

**Review by ssayer — APPROVED** [review]
No regressions with latest update

**Review by mulcahyk — APPROVED** [review]

**Review by holmesie — APPROVED** [review]
Looks good; Approved.

**Pull request merged**